catalog/source/iwc_manifest_to_workflows_yaml.py

```diff
@@ -0,0 +1,180 @@
+import argparse
+import json
+import os
+import subprocess
+from dataclasses import asdict, dataclass
+from typing import Any, Dict, List, Literal, Optional
+
+import requests
+import yaml
+
+URL = "https://iwc.galaxyproject.org/workflow_manifest.json"
+WORKFLOWS_PATH = "workflows.yml"
+CATEGORIES = Literal[
+    "VARIANT_CALLING",
+    "TRANSCRIPTOMICS",
+    "REGULATION",
+    "ASSEMBLY",
+    "GENOME_COMPARISONS",
+    "PROTEIN_FOLDING",
+    "OTHER",
+]
+DOCKSTORE_COLLECTION_TO_CATEGORY = {
```

Comment:
can we/ do we verify somewhere that all the categories listed in the categories yml have a mapping? or do we want to try to do that is i guess the first q..

Reply:
Put it in a file and read it from there ? I don't think this needs to be part of this PR.